03 Misc/09 Right pad library/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "right-pad",
+  "version": "1.0.0",
+  "description": "A library to append text",
+  "main": "dist/rightpad.js",
+  "types": "dist/rightpad.d.ts",
```

Comment:
I'm asking because I remember that I used `typings` field instead of `types` for same functionality. Btw, nice work